package.json

```diff
@@ -9,10 +9,14 @@
   "scripts": {
     "lint": "eslint --rulesdir=./node_modules/eslint-config-tjw-jest/custom-eslint-rules --ext .js --config=.eslintrc.cjs .eslintrc.cjs index.js library.js tests",
     "fix": "eslint --fix --rulesdir=./node_modules/eslint-config-tjw-jest/custom-eslint-rules --ext .js --config=.eslintrc.cjs .eslintrc.cjs index.js library.js tests",
-    "test": "vitest run --pool=forks"
+    "test": "vitest run --pool=forks --coverage"
```

Comment:
Added a coverage check, I don't really see this library as strictly needing 100% test coverage, but having this at least lets us keep better track going forward as it will display in GHA.